frontend/device/devicelistener.lua

```diff
@@ -20,6 +20,15 @@ function DeviceListener:onToggleNightMode()
     G_reader_settings:saveSetting("night_mode", not night_mode)
 end
 
+function DeviceListener:onSetNightMode(night_mode_on)
+    local night_mode = G_reader_settings:isTrue("night_mode")
+    if (night_mode_on and not night_mode) or ( not night_mode_on and night_mode) then
```

Comment:
That space on purpose? :-)
```suggestion
    if (night_mode_on and not night_mode) or (not night_mode_on and night_mode) then
```